internal/git/commands.go

```diff
@@ -519,11 +520,26 @@ func (self *Commands) HasLocalBranch(runner gitdomain.Runner, name gitdomain.Loc
 	return runner.Run("git", "show-ref", "--quiet", "refs/heads/"+name.String()) == nil
 }
 
-func (self *Commands) HasMergeInProgress(runner gitdomain.Runner) bool {
-	err := runner.Run("git", "rev-parse", "-q", "--verify", "MERGE_HEAD")
+func (self *Commands) HasMergeInProgress(querier gitdomain.Querier) bool {
+	_, err := querier.Query("git", "rev-parse", "-q", "--verify", "MERGE_HEAD")
```

Comment:
If you don't use the output returned by `querier.Query`, then you can also keep this as a `runner.Run`.

Reply:
Since I need to call it from `RepoStatus` which only has a `Querier`, it was easier to make this function also take a `Querier`.

Reply:
Okay, thanks for the background. Good as is then.

Reply:
On second thought... Can you update `RepoStatus` to receive a [RunnerQuerier](internal/git/gitdomain/runner.go) instead of a `Querier`? Then you should be able to call `HasMergeInProgress` with a `Runner`. `RunnerQuerier` implements both the `Runner` and the `Querier` interface.

Reply:
https://github.com/git-town/git-town/pull/4755